build/commands/lib/logging.js

```diff
@@ -142,9 +142,24 @@ function allPatchStatus(allPatchStatus, patchGroupName) {
       console.log(chalk.green(`${successfulPatches.length} successful patches:`))
       successfulPatches.forEach(logPatchStatus)
     }
+
     if (failedPatches.length) {
-      console.log(chalk.red(`${failedPatches.length} failed patches:`))
-      failedPatches.forEach(logPatchStatus)
+      if (printPatchFailuresInJson) {
+        console.log(
+            chalk.red(`${failedPatches.length} Failed patches json breakdown:`))
+        const patchFailuresOutput = failedPatches.map(({path, patchPath}) => {
+          return {
+            patchPath:
+              patchPath.substring(patchPath.indexOf('brave/patches')),
```

Comment:
can we use `path.join(config.braveCoreDir, 'patches')` instead of hardcoding?

Reply:
Done. To clarify, I was actually trimming the path, to make it relative.

Reply:
Actually I think this should be reverted to how it was, because what it was actually saying was `home/user/path/to/project/src/brave/patches/chrome-android-java-src-org-chromium-chrome-browser-sync-settings-ManageSyncSettings.java.patch` -> `brave/patches/chrome-android-java-src-org-chromium-chrome-browser-sync-settings-ManageSyncSettings.java.patch`